Sources/Orbit/Foundation/Typography/Font.swift

```diff
@@ -1,52 +1,70 @@
 import UIKit
 import SwiftUI
 
+var orbitFontNames: [Font.Weight: String] = [:]
+
 public extension Font {
 
+    /// Fonts used for rendering text in Orbit.
+    static var orbitTextFonts: [Font.Weight: URL?] = [
```

Comment:
To match `orbitFontNames`, this could be just `orbitFonts`?

Reply:
I suppose, I wanted to differentiate it from the icon font, but it's not necessary.

Reply:
Updated.